Gruntfile.js

```diff
@@ -96,14 +96,18 @@ module.exports = function(grunt) {
         options: {
           title: 'Angular-Patternfly Documentation',
           dest: 'dist/docs',
-          scripts: ['lib/patternfly/components/jquery/jquery.js',
+          scripts: ['lib/patternfly/components/jquery/dist/jquery.js',
             'lib/patternfly/components/bootstrap/dist/js/bootstrap.js',
-            'lib/patternfly/components/bootstrap-select/bootstrap-select.js',
+            'lib/patternfly/components/bootstrap-combobox/js/bootstrap-combobox.js',
+            'lib/patternfly/components/bootstrap-select/js/bootstrap-select.js',
+            'lib/patternfly/components/bootstrap-treeview/src/js/bootstrap-treeview.js',
+            'lib/patternfly/components/c3/c3.js',
+            'lib/patternfly/components/d3/d3.js',
```

Comment:
These were added in PF 1.3.0 so we are playing catchup with the includes since this project hasn't been upgraded in a bit. Combobox and treeview were also added a bit back but.

Granted you don't need them to run angular-pf as of yet but it if you want the docs to explain what includes PF needs then it is nice to have them here. So from my perspective it is a nice to have.